src/electron/views/index.html

```diff
@@ -10,6 +10,7 @@
         <link rel="stylesheet" type="text/css" href="./device-connect-view/styles/default/device-connect-view.css" />
         <link rel="stylesheet" type="text/css" href="../../electronBundle/renderer.css" />
         <title>Accessibility Insights for Mobile</title>
+        <script src="../../insights.config.js"></script>
```

Comment:
This is needed to attach the `appinsights` config on global `windows`. Ai-web uses `insights.html` to drive this; but since i did not need any thing other this file; decided to load this with `index.html`.